plugins-builtin/src/main/scala/za/co/absa/enceladus/plugins/builtin/common/mq/kafka/KafkaConnectionParams.scala

```diff
@@ -48,6 +49,8 @@ object KafkaConnectionParams {
   def fromConfig(conf: Config, clientIdKey: String, topicNameKey: String): KafkaConnectionParams = {
     validate(conf, clientIdKey, topicNameKey)
 
+    SecureKafka.setSecureKafkaProperties(conf)
```

Comment:
Is this still needed here if used in the `main` functions?

Reply:
Yeah, I guess not. I will remove it.

More to it. This initialization affects the whole application and all frameworks that use SSL, so it is not the place to do it in plugins.

Reply:
But also there is a rationale to keep it. The method is idempotent. If the system properties are already set, it won't overwrite them.
But I guess these plugins can be used only in `spark-jobs`, so removing it is still the better option.